i18n/locales/en.ts

```diff
@@ -1063,8 +1063,22 @@ const translations = {
     },
     paragraph_one:
       'Vanderpoole is out to get you now that you’ve exposed him as a fraud. But despite that, plenty of scared people continue to cling to the myth he created around himself, his family, and their supposed ancestry. Times are scary, and people need a hero. Unfortunately, for many, he’s the best they’ve got.',
-    in_out_one: {
+    in_out_two: {
       title: 'The ins and outs',
```

Comment:
You'll need to adjust the metadata titles to match this now being under in_out_two